build/SignToolData.json

```diff
@@ -0,0 +1,66 @@
+{
+  "sign": [
```

Comment:
Do you know why GitHub isn't seeing this as a rename? Are there actually changes to every line?

Reply:
not sure why it isn't, there should only be a couple modified lines (where I added quotations around some of the names)

Reply:
Are there unnecessary whitespace differences?